core/pom.xml

```diff
@@ -296,21 +285,25 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>org.springframework.security</groupId>
-      <artifactId>spring-security-acl</artifactId>
-      <version>${spring-security.version}</version>  <!-- see maven-parent-pom -->
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-aop</artifactId>
       <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.security</groupId>
+      <artifactId>spring-security-core</artifactId>
       <exclusions>
```

Comment:
If this artifact is just needed during tests then scope should be test `<scope>test</scope>`.

Reply:
Needed also for compilation, removed commentary